src/validation.cpp

```diff
@@ -5307,10 +5300,17 @@ static ChainstateManager::Options&& Flatten(ChainstateManager::Options&& opts)
     return std::move(opts);
 }
 
-ChainstateManager::ChainstateManager(Options options) : m_options{Flatten(std::move(options))} {}
+ChainstateManager::ChainstateManager(Options options)
+    : m_script_check_queue{/*nBatchSizeIn=*/128},
+      m_options{Flatten(std::move(options))}
+{
+    m_script_check_queue.StartWorkerThreads(m_options.worker_threads_num);
```

Comment:
Previously this was conditional on `worker_threads_num > 0` (since `StartScriptCheckWorkerThreads` would not be called unless `script_threads >= 1`). I think this is okay, since in that case the function only sets some variables that will never be used and skips over a loop.

Reply:
This code will gone in the "refactor: Make `CCheckQueue` constructor start worker threads" commit.